admin_guide/topics/creating-primed-images-cluster-auto-scaler.adoc

```diff
@@ -37,6 +37,9 @@ endif::[]
 ansible_ssh_user=ec2-user
 openshift_clusterid=mycluster
 ansible_become=yes
+openshift_cloudprovider_kind=aws
```

Comment:
I think that it might be useful to include some call outs for these three lines, outlining what they do. I have seen similar examples in other assemblies in this book.

Reply:
Thanks, Paul. I have added this enhancement.